CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.10.0
+### Added
+* CLI: Added `libfuzzer merge` job template, which enables running performing libfuzzer input minimization as a batch operation.  [#282](https://github.com/microsoft/onefuzz/pull/282)
+* CLI/Service: Added the instance specific Application Insights telemetry key to `onefuzz info get`, which will enable logging to the instance specific application insights from the SDK.  [#353](https://github.com/microsoft/onefuzz/pull/353)
+* Agent: Added support for parsing AddressSanitizer `CHECK failed` entries, which can occur during large amounts of memory corruption.  [#358](https://github.com/microsoft/onefuzz/pull/358)
+* Agent/Service: Added support for parsing the `ASAN_OPTIONS` output from `print_scariness`.  [#359](https://github.com/microsoft/onefuzz/pull/359)
+
+### Changed
+* Agent: Mark tasks as failed if the application under test generates an ASAN log file that the agent is unable to parse.  [#351](https://github.com/microsoft/onefuzz/pull/351)
+* Agent: Updated the `libfuzzer_merge` task to pre-existing inputs in a single pass.  [#282](https://github.com/microsoft/onefuzz/pull/282)
+* CLI: Clarified the error messages when prefix-expansion fails.  [#342](https://github.com/microsoft/onefuzz/pull/342)
+* Service: Rendered `pydantic` models as JSON when logging to remove `error=None` from showing up in the error logs.  [#350](https://github.com/microsoft/onefuzz/pull/350)
```

Comment:
```suggestion
* Service: Rendered `pydantic` models as JSON when logging to prevent `error=None` from showing up in the error logs.  [#350](https://github.com/microsoft/onefuzz/pull/350)
```